tarorpc/taro.proto

```diff
@@ -628,3 +635,21 @@ message SendAssetResponse {
 
     int64 total_fee_sats = 5;
 }
+
+message SubscribeSendAssetEventNtfnsRequest {
+}
+
+message SendAssetEvent {
+    oneof event {
+        // An event which indicates that a send state is about to be executed.
+        ExecuteSendStateEvent execute_send_state_event = 1;
+    }
+}
+
+message ExecuteSendStateEvent {
+    // Execute timestamp.
+    int64 timestamp = 1;
+
+    // The send state that is about to be executed.
+    string send_state = 2;
```

Comment:
Seems like this should instead be another enum? Otherwise not sure what the actual strucutre would be. 

Or was this meant to be `SendAssetEvent` here?

Reply:
`ExecuteSendStateEvent` is one type of event that can be streamed. That event type is published before the state machine executes a state.

The `SendAssetEvent` message wrapps all streamed event types.

Reply:
Sounds like in the future we'd use this nesting to more easily extend to other event types from other subsystems?

Was a bit confused re: the extra nesting.

Reply:
> we'd use this nesting to more easily extend to other event types

Yes, that's what I was going for.